app/controllers/games_controller.rb

```diff
@@ -0,0 +1,19 @@
+class GamesController < ApplicationController
+  before_action :authenticate_user!, only: %i[new create show]
+
+  def new; end
+
+  def create
+    @game = Game.new(game_params)
+    @game.save
```

Comment:
`@game = Game.create(game_params)`

Is the same as using `new` and then `save`